### PR TITLE
[CIAPP-5685] add explanation on flaky test attributes

### PR DESCRIPTION
**Pull request by alxch-**
### What does this PR do?

This PR adds a table detailing attributes that are added to flaky tests.

### Motivation
We have a few customers that reached out to ask for more help on how to create flaky test monitors and dashboards.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by rtrieu** [comment]
Created [DOCS-4696](https://datadoghq.atlassian.net/browse/DOCS-4696) for the docs team, as I am not confident on the topic.

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 1]
```diff
@@ -76,5 +76,17 @@ Failed test runs that were known to be flaky as per the definition above are dis
 
 {{< img src="ci/known-flaky-failed-tests.png" alt="CI Tests Branches view with a branch selected and a text box in the Failed column displaying 1 tests failed and 1 known flaky" style="width:100%;">}}
 
+
+## Flaky tests attributes
+
+
+Test runs that are flaky, new flaky or known flaky contain attributes that can be used to filter queries or build monitors.
```
Comment (fermayo) [inline content/en/continuous_integration/guides/flaky_test_management.md:83]:
```suggestion
Test runs that are flaky, new flaky or known flaky contain attributes that can be used to filter queries or create dashboards and monitors.
```

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 2]
```diff
@@ -76,5 +76,17 @@ Failed test runs that were known to be flaky as per the definition above are dis
...
+
+| Name        | Attribute                   | Description                                                                                                                                                                                                                                                                                                  |
+|-------------|-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Flaky       | `@test.is_flaky:true`       | Test runs that exhibit a flaky behavior in the current commit: a previous test run was received for the same test and passed (resp. failed) whereas this test run fails (resp. passes). The attribute is only present the first time a test is detected flaky in a commit.                                   |
```
Comment (fermayo) [inline content/en/continuous_integration/guides/flaky_test_management.md:87]:
```suggestion
| Flaky       | `@test.is_flaky:true`       | The test run has flaked in the commit. Either this test run has failed and there was a previous test run for the same test and commit that passed, or this test run passed and there is a previous failure. This attribute is only present the first time a test is detected flaky in a commit. |
```

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 3]
```diff
@@ -76,5 +76,17 @@ Failed test runs that were known to be flaky as per the definition above are dis
...
+| New Flaky   | `@test.is_known_flaky:true` | Test runs that exhibit a flaky behavior and didn’t previously exist in the Flaky Tests table for the current branch or default branch of the repository. When a test run has this attribute, it means that it has never been detected as flaky so this commit is introducing a new flaky test in the branch. |
```
Comment (fermayo) [inline content/en/continuous_integration/guides/flaky_test_management.md:88]:
```suggestion
| New Flaky   | `@test.is_new_flaky:true` | The test run has flaked in the commit (according to the "Flaky" definition above) and didn’t previously exist in the "Flaky tests" table for the branch of the test run or default branch of the repository. This also means that the test has been added to the "Flaky tests" table. |
```

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 4]
```diff
@@ -76,5 +76,17 @@ Failed test runs that were known to be flaky as per the definition above are dis
...
+| Known Flaky | `@test.is_new_flaky:true`   | **Failing** tests runs that that previously exhibited flaky behavior on the current or default branch of the repository. When a test run has this attribute, it means that the test was detected flaky in the past and that this failure might not be the result of a change in this commit.                 |
```
Comment (fermayo) [inline content/en/continuous_integration/guides/flaky_test_management.md:89]:
```suggestion
| Known Flaky | `@test.is_known_flaky:true`   | The test run has flaked in the commit (according to the "Flaky" definition above) and it was already present on the "Flaky test" table of either the branch of the test run or the default branch of the repository. This means that the test was detected flaky in the past and that this failure might not be the result of a change in this commit. |
```

**Review by fermayo — APPROVED** [review]

File: content/en/continuous_integration/guides/flaky_test_management.md [hunk 5]
```diff
@@ -76,5 +76,17 @@ Failed test runs that were known to be flaky as per the definition above are dis
...
+| Flaky       | `@test.is_flaky:true`       | The test run has flaked in the commit. Either this test run has failed and there was a previous test run for the same test and commit that passed, or this test run passed and there is a previous failure. This attribute is only present the first time a test is detected flaky in a commit. |
+| New Flaky   | `@test.is_new_flaky:true` | The test run has flaked in the commit (according to the "Flaky" definition above) and didn’t previously exist in the "Flaky tests" table for the branch of the test run or default branch of the repository. This also means that the test has been added to the "Flaky tests" table. |
+| Known Flaky | `@test.is_known_flaky:true`   | The test run has flaked in the commit (according to the "Flaky" definition above) and it was already present on the "Flaky test" table of either the branch of the test run or the default branch of the repository. This means that the test was detected flaky in the past and that this failure might not be the result of a change in this commit. |
```
Comment (jtappa) [inline content/en/continuous_integration/guides/flaky_test_management.md:89]:
```suggestion
| Known Flaky | `@test.is_known_flaky:true`   | The test run has flaked in the commit (according to the "Flaky" definition above) and it was already present on the "Flaky test" table of either the branch of the test run or the default branch of the repository. This means that the test was detected as flaky in the past, and that this failure may not be the result of a change in this commit. |
```

**Review by jtappa — APPROVED** [review]

**Pull request merged**